### ISSUE TITLE
Atom feed results for Changelog List page is incorrect

### ISSUE THREAD

**Issue by luzpaz** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#1231]
## Description

Page URL: https://changelog.qgis.org/en/qgis/version/list/  
The feed URL (top right corner, see screenshot) does not return the correct context. 
![Screenshot_20240310_101749](https://github.com/qgis/QGIS-Website/assets/4140247/e5e8a950-b1ab-448b-84ef-3c8f021a7c81)

Here's the feed URL: https://changelog.qgis.org/en/qgis/atom/latest-version/

**Comment by DelazJ** [comment]
I don't think this issue report belongs to this repository. @timlinux what would be the correct one? And mind transferring? Thx.

**Comment by rduivenvoorde** [comment]
Should be https://github.com/kartoza/prj.app but I'm not able to transfer
